network.js

```diff
@@ -52,9 +51,8 @@ module.exports = {
   callFetch(dest, opts) {
     return fetch(dest, opts);
   },
-  downloadFile(originalUrl) {
-    var resolve, reject,
-    savePath = path.join(platform.getTempDir(), url.parse(originalUrl).pathname.split("/").pop());
```

Comment:
@fjvallarino can't we leave this here as a fallback in case the savePath param isn't provided?